Kernel/Arch/riscv64/Timer.cpp

```diff
@@ -31,7 +32,7 @@ NonnullLockRefPtr<Timer> Timer::initialize()
     return timer;
 }
 
-u64 Timer::microseconds_since_boot()
+u64 Timer::current_time()
```

Comment:
I renamed it to `current_ticks` now, hopefully that makes it more clearer.
I just called it `current_time`, because that is what RISC-V calls the CSR.